crates/fluvio-protocol-derive/src/ast/enum.rs

```diff
@@ -60,28 +64,31 @@ pub(crate) struct EnumProp {
     pub tag: Option<String>,
     pub discriminant: Option<DiscrimantExpr>,
     pub kind: FieldKind,
+    pub min_version: Option<PropAttrsType>,
```

Comment:
I added them because I saw that here it's trying to use them but they were not existent. Should I just remove them even though this Enum is trying to use them?

https://github.com/infinyon/fluvio/blob/0a4e03406db93c651fc3dbd1e0657a57281ab302/crates/fluvio-spu-schema/src/server/smartmodule.rs#L88-L104

Reply:
It won't change anything if I remove them, it will still work and still won't do anything of course :D

Reply:
Yes, so there are a couple of options here: 
1. make it work
2. leave as is and create a follow-up issue
3. remove wrong occurrences from the existing code, add compilation warning and create a follow-up ticket
Pick any.
Adding the unworking code to the unworking code is not a proper solution :)

Reply:
I guess I lied a bit here :D Because of the new changes we are checking if attribute actually exists, so currently because there are some places where there are enums with min_version/max_version then if we remove it from here it throws similar to this error "`attribute min_version unrecognized`". So maybe lets leave it like that at the moment and with a new issue add the functionality for the enum versioning?

Reply:
Ok